types/meteor/meteor.d.ts

```diff
@@ -408,7 +416,7 @@ declare module 'meteor/meteor' {
             close: () => void;
             onClose: (callback: () => void) => void;
             clientAddress: string;
-            httpHeaders: Object;
+            httpHeaders: Record<string, string>;
```

Comment:
Headers are most likely `string | string[]`.

Reply:
That's what I would have guessed, but this comes from sockjs, which declares them to just be string: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b483f710a8923b277896fc36de4454752a7d24b6/types/sockjs/index.d.ts#L39-L41